ui/pages/bridge/utils/quote.ts

```diff
@@ -224,7 +236,7 @@ export const formatCurrencyAmount = (
   }
   if (precision === 0) {
     if (amount.lt(0.01)) {
-      return `<${formatCurrency('0', currency, precision)}`;
+      return '<$0.01';
```

Comment:
Not a blocker, as this is a pre-existing issue, but this is not localized correctly. We should be using `Intl.NumberFormat` for all number/currency displays, passing in the currency symbol and locale so that it's rendered correctly.

Some locales use `,` instead of `.`, and of course currency symbols can differ (maybe this is always USD? 🤔  But still)

Reply:
will note this as a fast-follow task!